news/2021-06-02-new-featured-artist-fleshgod-apocalypse.md

```diff
@@ -0,0 +1,81 @@
+
+**19** metal headbangers (including those above) are ready for you to try your hand at mapping from [**Fleshgod Apocalypse**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/170)!
+
+Check it all out:
+
+### The Deceit
+
+Try [this 250 BPM stream-heavy map](https://osu.ppy.sh/beatmapsets/367631) hosted by [Mazzerin](https://osu.ppy.sh/beatmapsets/991300)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### The Violation
+
+Or try [this 270 BPM stream-heavy map](https://osu.ppy.sh/beatmapsets/1198838) also hosted by [Mazzerin](https://osu.ppy.sh/beatmapsets/991300)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+Players of all modes can attempt a combination of both *The Deceit* and *The Violation* through [this loved map](https://osu.ppy.sh/beatmapsets/367631)!
+
+### In Aeternum
+
+Give [this osu! map](https://osu.ppy.sh/beatmapsets/1261793) hosted by [Plaudible](https://osu.ppy.sh/users/7149815) or [this osu!taiko map](https://osu.ppy.sh/beatmapsets/1249644) by [Heaxys](https://osu.ppy.sh/users/5671417) a play too!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### The Fool
+
+Play through any of these maps:
+
+- [Loved osu!catch difficulty spread](https://osu.ppy.sh/beatmapsets/445798) hosted by [Razor Sharp](https://osu.ppy.sh/users/3414261)
+- [Ranked osu!mania difficulty spread](https://osu.ppy.sh/beatmapsets/1301878) hosted by [Mipha-](https://osu.ppy.sh/beatmapsets/1301878)
+- [Loved osu!taiko map](https://osu.ppy.sh/beatmapsets/442451) hosted by [LZD](https://osu.ppy.sh/users/224335)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### The Hypocrisy
+
+Slam your piano keys to [this osu!mania map](https://osu.ppy.sh/beatmapsets/397784) by [Phil](https://osu.ppy.sh/users/3191489)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### And The Vulture Beholds
+
+Or whack your drums to [this osu!taiko map](https://osu.ppy.sh/beatmapsets/987536) by [GoldenWolf](https://osu.ppy.sh/users/1612624)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+---
+
+As per usual, the rest of what you're looking for can be found on [**Fleshgod Apocalypse**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/170), primed for mapping thanks to our ultra-convenient beatmap templates. It's about time your maps reached the infamous black difficulty icon, right?
+
+And that's all for now! Sticking to the trend of artists who've grown in the osu! community through [Project Loved](/wiki/Project_Loved), our next artist is one you might be familiar with if you're an avid fan of unbelievable [WhiteCat](https://osu.ppy.sh/users/4504101) and [Vaxei](https://osu.ppy.sh/users/4787150) DT plays.
```

Comment:
hmm, i'm not entirely sure about the context here but i think in general the last sentence on this paragraph here would be better off to be written like this tho :

`...if you're an avid fan of unbelievable DT plays from the likes of [WhiteCat](https://osu.ppy.sh/users/4504101) and [Vaxei](https://osu.ppy.sh/users/4787150).`

Reply:
verytrue